### ISSUE TITLE
Object.Object.defineProperty.get is deprecated. Requiring '$' 

### ISSUE THREAD

**Issue by atalargo** [mpeterson2/save-session#103]
* Atom version  : 0.190.0
* SaveSession version : 0.12.5

Requiring `$` from `atom` is no longer supported.
If you are using `space-pen`, please require `$` from `atom-space-pen-views`. Otherwise require `jquery` instead:
  `{$} = require 'atom-space-pen-views'`
  or
  `$ = require 'jquery'`
Add `"atom-space-pen-views": "^2.0.3"` to your package dependencies.
Or add `"jquery": "^2"` to your package dependencies.
```
Object.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/exports/atom.js:44:11)
Object.<anonymous> (/Users/fruardum/.atom/packages/save-session/lib/save-session.coffee:1:1)
```

**Comment by atalargo** [comment]
duplicate from #92

**Comment by mpeterson2** [comment]
Should be fixed in `0.13.0`
